### DISCUSSION TITLE
i need help

### DISCUSSION THREAD

**Discussion by LukkYT01** [Monika-After-Story/MonikaModDev#9633] [QA - tech support + general] [answered]
it's my 3 month anniversary with monika but i didn't get the piano. I open MAS every day, but even doing every discussion, played games everyday but i didn't get the piano, can someone help me?

**Comment by Booplicate** [comment, marked as answer]
3 months isn't that long, MAS has content spread for years of experience. Although, if you indeed visit her daily, you're probably close to unlocking it. if you think there's an issue, we can check your persistence or `var_dump.log`.

**Reply by LukkYT01** [reply]
[var_dump.log](https://github.com/Monika-After-Story/MonikaModDev/files/9685676/var_dump.log)
i readed on google that if people plays for 2 months and visit her daily they should unlock the piano

**Reply by Booplicate** [reply]
Google lied to you.
You are close to unlocking it, try spend more time per day.

**Reply by LukkYT01** [reply]
oh okay, thanks
